tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -177,32 +177,34 @@ Docker volume create with possibly invalid name
     Should Be Equal As Strings  ${output}  Error response from daemon: volume name "test???" includes invalid characters, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed
 
 Docker volume verify anonymous volume contains base image files
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name verify-anon-1 jakedsouza/group-1-19-docker-verify-volume-files:1.0 ls /etc/example
-    Should Be Equal As Integers  ${rc}  0
-    Should Contain  ${output}  thisshouldexist
-    Should Contain  ${output}  testfile.txt
-
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name verify-anon-2 jakedsouza/group-1-19-docker-verify-volume-files:1.0 cat /etc/example/testfile.txt
-    Should Be Equal As Integers  ${rc}  0
-    Should Contain  ${output}  TestFile
-
-Docker volume verify named volume contains base image files
-	${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name verify-named-1 -v test15:/etc/example jakedsouza/group-1-19-docker-verify-volume-files:1.0 cat /etc/example/testfile.txt
-    Should Be Equal As Integers  ${rc}  0
-    Should Contain  ${output}  TestFile
+    ${status}=  Get State Of Github Issue  7365
```

Comment:
@mhagen-vmware identified `1-25: Container to container traffic via VCH public interface` as the test that was failing repeatedly for #7365.

Given it appears to be specific to cleanup of a container with 4 network endpoints defined I do not see why this test would be hitting it. Please revisit the diagnosis that cause this to be commented out with that as the blocker.
If it was due to similar docker client level _symptoms_ then you may be seeing a port layer panic _for other reasons_ and we need a separate issue to investigate.

Reply:
@mhagen-vmware ?

Reply:
I am working on it, but isn't gating this PR.